docs/bin/bundle.sh

```diff
@@ -35,7 +35,10 @@ function _set_sdk_version {
   local -r dir="$1"
   export SDK_VERSION=$sdk_version
   # we only want to update the first occurrence of <version>, the one belonging the parent-pom
+  echo $dir/pom.xml
   awk '/<version>[^<]*<\/version>/ && !subyet {sub("<version>[^<]*<\/version>", "<version>"ENVIRON["SDK_VERSION"]"</version>"); subyet=1} 1' $dir/pom.xml > temp && mv temp $dir/pom.xml
+  echo "Updated SDK_VERSION in $dir/pom.xml to $SDK_VERSION"
+  echo $dir/pom.xml
```

Comment:
don't you want to `cat $dir/pom.xml | grep 'version'`?

Reply:
pom.xml should be relatively small, but yeah I could improve it